### PR TITLE
hardshrink and Hardshrink doc

### PR DESCRIPTION
**Pull request by zhupengyang**
image
image

### CODE REVIEW

File: doc/fluid/api_cn/nn_cn/activation_cn/Hardshrink_cn.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.. code-block:: python
+
+    import paddle
+    import numpy as np
+
+    paddle.disable_static()
+
+    x = paddle.to_variable(np.array([-1, 0.3, 2.5]))
+    hd = paddle.nn.Hardshrink()
```
Comment (T8T9) [inline doc/fluid/api_cn/nn_cn/activation_cn/Hardshrink_cn.rst:42]: 用 m = paddle.nn.Hardshrink() ?

Comment (zhupengyang) [reply doc/fluid/api_cn/nn_cn/activation_cn/Hardshrink_cn.rst]: Done

**Comment by T8T9** [comment]
LGTM

**Review by swtkiwi — APPROVED** [review]
LGTM

**Review by hong19860320 — APPROVED** [review]
LGTM

**Pull request merged**